src/components/DownloadDataModal.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState, useMemo } from 'react';
+
+import styled from 'styled-components/macro';
+import { theme } from '../constants/theme';
+import { withStyles } from '@material-ui/core/styles';
+import MuiDialogTitle from '@material-ui/core/DialogTitle';
+import MuiDialogContent from '@material-ui/core/DialogContent';
+import MuiDialogActions from '@material-ui/core/DialogActions';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import Button from '@material-ui/core/Button';
+import { Box, Dialog, useMediaQuery } from '@material-ui/core';
+import { DialogText } from '../styles/MermaidStyledComponents';
+import ContactIcon from '@material-ui/icons/Email';
+import CloudDownloadIcon from '@material-ui/icons/CloudDownload';
+import { protocolMethods, pluralizedProtocols } from '../constants/transect-protocols';
+import TooltipLayout from './TooltipLayout';
+
+const ContactIconWrapper = styled(ContactIcon)`
+  padding-right: 5px;
+  font-size: small;
+`;
+
+const CloudDownloadIconWrapper = styled(CloudDownloadIcon)`
+  padding-right: 5px;
+  font-size: small;
+`;
+
+const MermaidButton = withStyles({
+  root: {
+    margin: '4px',
+    color: theme.cardButton.color.mermaidWhite,
+    backgroundColor: theme.cardButton.bgColor,
+    '&:hover': {
+      backgroundColor: theme.cardButton.bgColor
+    }
+  }
+})(Button);
```

Comment:
Is this a one-off styling of this button? If not, is it worth it to make it a reusable component?